### DISCUSSION TITLE
Withdrawal of consent

### DISCUSSION THREAD

**Discussion by noah-felten** [maize-tech/laravel-legal-consent#9] [Q&A]
Dear Maize Team, 

thank you for this great package! I've got one question, is there an intended way for a user to withdraw consent to a document? I see there's no route defined for that case. My idea is to just delete the entry in legal_consents table which seems obvious to me, but mayber you have a better idea in mind :) 

Thank you and kind regards

Noah

**Comment by riccardodallavia** [comment]
HI @noah-felten and thanks for the suggestion!
It is indeed a really really nice feature idea: would you like to submit a pull request for this?
Without having given it much thought, I'd implement a new endpoint that allows the user to withdraw a specific consent or all the existing ones.
Thank you!
